blockchain/storage/leveldb/leveldb_test.go

```diff
@@ -37,3 +48,227 @@ func newStorage(t *testing.T) (storage.Storage, func()) {
 func TestStorage(t *testing.T) {
 	storage.TestStorage(t, newStorage)
 }
+
+func generateRandomByteSlice(count int) []byte {
+	s := make([]byte, count)
+	for i := 0; i < count; i++ {
+		s[i] = byte(rand.Int())
+	}
+
+	return s
+}
+
+func generateFullTx(nonce uint64, from types.Address, to *types.Address, value *big.Int, dynamic bool, v *big.Int) *types.Transaction {
```

Comment:
Double check, maybe some of these helper functions are already implemented, so they can be reused.

Reply:
There isn't anything exposed in common. we can do that in some other pr if necessary